### ISSUE TITLE
Instructions don't seem to be working

### ISSUE THREAD

**Issue by lokijota** [MicrosoftDocs/powerapps-docs#1984]
I've been using the technique documented here to integrate with Dynamics for 2 years. This however stopped working a couple of days ago, either because of a change on Dynamics or (more likely) Postman. Can you please verify with the latest version of Postman?

I'm getting a popup error with the following, when I try to get a token with OAUth 2.0:

```
Error
An error has occurred.Try this action again. If the problem continues, check the Microsoft Dynamics 365 Community for solutions or contact your organization's Microsoft Dynamics 365 Administrator. Finally, you can contact Microsoft Support.

Timestamp: 12/4/2020 7:51:59 PM UTC
Service Request Id: ddd7349f-0cbf-4552-a366-a9570948fb5a
```

in this URL: <https://xxxxx.crm.dynamics.com/_common/error/errorhandler.aspx?BackUri=&ErrorCode=&Parm0=%0d%0a%0d%0aError%20Details%3a%20An%20unhandled%20exception%20occurred%20during%20the%20execution%20of%20the%20current%20web%20request.%20Please%20review%20the%20stack%20trace%20for%20more%20information%20about%20the%20error%20and%20where%20it%20originated%20in%20the%20code.&RequestUri=%2fdefault.aspx%3fresponse_type%3dcode%26state%3d%26client_id%3d51f81489-12ee-4a9e-aaae-a2591f45987d%26scope%3d%26redirect_uri%3dhttps%253A%252F%252Foauth.pstmn.io%252Fv1%252Fcallback%2fdefault.aspx&user_lcid=1033>

And the details part says:

```
Tenant Id: 72f988bf-xxxxxxx
Url: https://xxxxx.crm.dynamics.com
Organization Id: aeae68a5-xxxxxxxxxxx
Type: Secondary
Unique name: xxxxxx
Created on: 7/23/2014 5:36:49 PM
State: Enabled
Security Group: 00000000-0000-0000-0000-000000000000
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 80534df2-e9da-fd0a-80bf-4bfc0dbb2fcc
* Version Independent ID: b8053ca7-e385-2f0b-06af-81b908f345a1
* Content: [Set up a Postman environment (Microsoft Dataverse for Apps) - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/setup-postman-environment#connect-with-your-common-data-service-environment)
* Content Source: [powerapps-docs/developer/common-data-service/webapi/setup-postman-environment.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/common-data-service/webapi/setup-postman-environment.md)
* Service: **powerapps**
* GitHub Login: @JimDaly
* Microsoft Alias: **jdaly**

**Comment by JimDaly** [comment]
Not long ago Postman shipped  version 7.35 which broke OAuth for everyone.
I know I had to completely uninstall the client, then install version 7.34
Details are [here](https://github.com/postmanlabs/postman-app-support/issues/9056#issuecomment-722320605)
But they seemed to have fixed it and I'm using version 7.36 now with no serious issue.

But your case may be different: 
The error in your URL (after decoding) looks like this:

```
https://xxxxx.crm.dynamics.com/_common/error/errorhandler.aspx?BackUri=&ErrorCode=&Parm0=

Error Details: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code.&RequestUri=/default.aspx?response_type=code&state=&client_id=51f81489-12ee-4a9e-aaae-a2591f45987d&scope=&redirect_uri=https://oauth.pstmn.io/v1/callback/default.aspx&user_lcid=1033
```

That redirect_uri doesn't look right, and the error is coming from the web application rather than from the API endpoint.

1. Make sure your environment variables look like this:
![PostmanEnvironment](https://user-images.githubusercontent.com/6353736/101210929-29efb100-362b-11eb-8628-624bad4076ef.png)
Note that the callback variable is 'https://callbackurl'.

2. Then check that the authorization tab uses those variables, Grant Type is Implicit.

![image](https://user-images.githubusercontent.com/6353736/101211052-64f1e480-362b-11eb-83b0-fddb9f11d95c.png)

Postman changed their UI so it is different from the topic. We can't always maintain this to keep up with every change PostMan does, but the core ideas are the same.

I've found that when creating a new tab, I have to reconfigure the Authorization each time. I don't think I needed to do that before..

**Comment by phecke** [comment]
@lokijota Thanks for your feedback. We are closing this issue because we are either unable to find relevant information or reproduce this. If you're still experiencing the issue described here, please reopen or create a new issue.
